documentation/blog/2023-06-22-svelte-dev-overhaul.md

```diff
@@ -0,0 +1,96 @@
+---
+title: 'svelte.dev: A complete overhaul'
+description: 'svelte.dev revamped, comes with accessibility fixes, new features and bottom navbar'
+author: Puru Vijay
+authorURL: https://puruvj.dev
+---
+
+It is June 2023, 4 years after svelte v3 was released. The then-practical-now-notorious single-pager website that folks are familiar with also came out then, and has stayed the same until now. Svelte's API surface has been increasing steadily and the single pager got bigger and bigger, to the point where it was becoming difficult to find things. The community has been asking for a revamp for a while now, and it's here now!
+
+Meet the new svelte.dev, a complete overhaul of the old website.
+
+## Multi-page docs
+
+The huge single-pager was a huge nuisance, in terms of finding stuff(Hullo there Ctrl+F), as well as just having low performance due to the sheer size of the page.
+
+We heard you! The docs are all split up into multiple pages now. And all pages are split into their sections in the sidebar. Wanna know how to use the `class:` directive, or wanna look up the syntax for `{#await}`? All these are just one click away under the `Template Syntax` heading in the sidebar.
```

Comment:
I think it's a bit hard to demonstrate the better organization with an example like this. I wouldn't necessarily think `class:` is obviously under `Template Syntax` without seeing the other headings and guessing which one is the best fit. If I knew I was looking for `class:` I'd probably just use search. Maybe better to just leave out the example
```suggestion
We heard you! The docs are all split up into multiple pages now and all pages list their sections in the righthand sidebar.
```